ansible_lab.yml

```diff
@@ -23,65 +22,29 @@
       tags:
         - baseline
 
-    - name: Backup existing SSH config
```

Comment:
Would like to retain because if users manually alter files, there will be no backup. Thus is better to be safe than sorry :)